src/components/RightSidebar/RightSidebar.vue

```diff
@@ -254,6 +292,30 @@ export default {
 			emit('show-settings')
 		},
 
+		async handleUpdateDescription(description) {
+			this.isDescriptionLoading = true
+			try {
+				await this.$store.dispatch('setConversationDescription', {
+					token: this.token,
+					description,
+				})
+				this.isEditingDescription = false
+			} catch (error) {
+				console.debug('Error while setting conversation description', error)
```

Comment:
console.error so it appears red and can be useful when people post errors in tickets